### PR TITLE
Maggie's first reflection

### PR DESCRIPTION
**Pull request by maggie-pbj**
@Osanchez-txabarri  and @qkrghdrms123 -- Writing about Rob Nixon's _Slow Violence_ and Jimmy Johns. Thanks!

### CODE REVIEW

**Comment by qkrghdrms123** [comment]
Hi, Jeffrey here.
First of all, I think you are far more better at md files than me :)
I think your file is nicely formatted and really organized.

I really am parallel to your point. I think there are a lot of things hidden under the wealth. Recently, I have watched a documentary about what is truly killing our marine life. I was able to understand how much of those plastic trash was made of and why the movements were only focusing on normal plastic trash: straws. I was shocked by how much of hidden truths there were behind the wealth and behind the who is funding. I think it is important to understand who is true cause of the plastic waste and how they are legally permitted to do so. (Just to comment, the plastic wastes were mostly from the fishermen; plus, massive fishing is destroying the marine ecosystem)

**Comment by Osanchez-txabarri** [comment]
Hi Maggie,

I agree with Jeffrey, you are much better than I at Md.

At the same time, I do really agree with you. It is sad and infuriating to see how these rich people who could actually do so much for the environment, spend their money creating greater harm. For another class, this semester I have been researching the lives of migrant women in big cities in South Africa and I can guarantee that animals and the environment are not the only ones who suffer the consequences of a world economy that values more than the money of some white rich people (not only American) than the lives of millions. since the 1990s South African cities have become the center of exploitation of women who leave their countries for a better opportunity and think that Johannesburg, for example, is just going to be a stop on their trip but consumes them inside. This is not to minimalize the importance of the destruction of the environment and nature, but rather to see how far the inequity of capitalism goes.

File: Slow Violence Reflection.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+I was resentful because men like Jimmy are men that could make the environment better but do the oppposite. Men like Jimmy are men that kill elephants and make billions. However, now with the context of Nixon’s chapter, I can begin to understand that slow violence is a sequence of events, not just one single nuanced action. Jimmy shot that elephant fast, and it hit the ground fast. I assumed he was a villain, fast. I kicked myself for working at a sandwich shop he started, fast. But then I happily deposited my checks, again fast. But holistically, this process of killing, judgement, and acceptance is slow and violent. Every number 10 Hunter’s Club isn’t a blow necessarily to South African local people, but it is a buffer between us and the reality of the nature and environment we are far from understanding.
```
Comment (Osanchez-txabarri) [inline Slow Violence Reflection.md:23]: I loved this paragraph, by the way. An amazing way of putting how the fast violence of our daily lives erases the importance of the slow violence of capitalism.

Comment (maggie-pbj) [reply Slow Violence Reflection.md:23]: Thank you, Olatz! I had a lot of pent-up rage of the fast violence within the fast food industry exposed me to: a sore back, unresolved sexual harassment, a cut on my tumb, red hands from an ingredient inside the gloves, a broken spirit, etc. However, reading Nixon's article and taking a look behind the scenes of the founder of the corporation I worked for, my understanding of the violence was compounded. Suddenly, Jimmy Johns sandwiches represented a plate full of issues much larger and harder to swallow than my month of poor me's.

**Comment by maggie-pbj** [comment]
> Hi, Jeffrey here.
> First of all, I think you are far more better at md files than me :)
> I think your file is nicely formatted and really organized.
> 
> I really am parallel to your point. I think there are a lot of things hidden under the wealth. Recently, I have watched a documentary about what is truly killing our marine life. I was able to understand how much of those plastic trash was made of and why the movements were only focusing on normal plastic trash: straws. I was shocked by how much of hidden truths there were behind the wealth and behind the who is funding. I think it is important to understand who is true cause of the plastic waste and how they are legally permitted to do so. (Just to comment, the plastic wastes were mostly from the fishermen; plus, massive fishing is destroying the marine ecosystem)

Thank you for your review, Jeffrey. I am not sure how I ended up in a place to have my markdown complimented, but I am happy for it! I appreciate your reference to the marine life documentary, and my ANTH professor and senior thesis advisor even talked about that in my Visual Anthropology classes during the discussion of lenses and cinematic plots in documentaries. She brought up the point that the documentary was created by a white filmmaker with little understanding of cultural contexts of the environmentally harmful fishing methods. It almost feels like a catch-22, because as being involved in the digital humanities, we are responsible for employing a cultural relativist and activist view, which can collide on the grounds of multiple environments inhabited and sometimes exploited by multiple groups.

**Comment by maggie-pbj** [comment]
> Hi Maggie,
> 
> I agree with Jeffrey, you are much better than I at Md.
> 
> At the same time, I do really agree with you. It is sad and infuriating to see how these rich people who could actually do so much for the environment, spend their money creating greater harm. For another class, this semester I have been researching the lives of migrant women in big cities in South Africa and I can guarantee that animals and the environment are not the only ones who suffer the consequences of a world economy that values more than the money of some white rich people (not only American) than the lives of millions. since the 1990s South African cities have become the center of exploitation of women who leave their countries for a better opportunity and think that Johannesburg, for example, is just going to be a stop on their trip but consumes them inside. This is not to minimalize the importance of the destruction of the environment and nature, but rather to see how far the inequity of capitalism goes.

Hi again, Olatz, thank you again for another comprehensive and insightful comment about wealth distribution and its effect on nature, not to mention different cultural understanding and representations of nature. I appreciate your added perspective of the migrant South African women in urban settings, as we often associate nature with wildlife when human populations are a part of those ecosystems. It's interesting how very little nature documentaries, for example, focus on the outcomes playing out in the health of floral and fauna rather than the mental, physical, emotional, and spiritual well-being of the increasingly exploited and entangled people.

**Pull request merged**